madara/crates/client/block_production/src/lib.rs

```
@@ -131,12 +129,10 @@ impl<Mempool: MempoolProvider> BlockProductionTask<Mempool> {
     /// as was done before.
     pub async fn close_pending_block(
         backend: &MadaraBackend,
-        importer: &BlockImporter,
         metrics: &BlockProductionMetrics,
     ) -> Result<(), Cow<'static, str>> {
```

why not return an anyhow error?

---

good question, it's changed in a later pr (i didnt want to change this crate too much)